src/OracleService/Protocols/OracleHttpsProtocol.cs

```diff
@@ -94,7 +94,10 @@ public async Task<(OracleResponseCode, string)> ProcessAsync(Uri uri, Cancellati
                 return (OracleResponseCode.ResponseTooLarge, null);
 
             var encoding = GetEncoding(message.Content.Headers);
-            return (OracleResponseCode.Success, encoding.GetString(buffer, 0, read));
+            if (!encoding.Equals(Encoding.UTF8))
+                return (OracleResponseCode.ContentTypeNotSupported, null);
```

Comment:
```suggestion
                return (OracleResponseCode.Error, null);
```

Reply:
Apply this one and merge? Anything else holding us on this PR?

Reply:
Everything else LGTM (except this comment).